src/coreclr/vm/peassembly.h

```diff
@@ -170,8 +127,22 @@ class PEFile
     const SString &GetModuleFileNameHint();
 #endif // DACCESS_COMPILE
 
+    LPCWSTR GetPathForErrorMessages();
+
+    // This returns a non-empty path representing the source of the assembly; it may
+    // be the parent assembly for dynamic or memory assemblies
+    const SString& GetEffectivePath();
+
+    // Codebase is the fusion codebase or path for the assembly.  It is in URL format.
```

Comment:
Quick search finds 48 references to "fusion" in the comments. It is not always clear how to re-word them, and the old context may still be useful while refactoring.  
I think it may be worth to revisit "fusion" comments a bit later. Fortunately it is such an easy thing to find.